main.js

```diff
@@ -543,7 +554,9 @@ function parseJSONString(jsonString, column, row) {
 
 //Generates the HTML for a cell in the grid
 function generateItemDOMFragment(id) {
-    let jobsDropDownHtml = createDropDownHTML(jobsListName, jobsList, id, jobsPlaceholderMessage);
+    // TODO: variable-ify these strings
```

Comment:
Does this TODO need to be completed before this PR goes in?

Reply:
Nope